acc-testcases/resources/lb_monitor.yaml

```diff
@@ -0,0 +1,106 @@
+vars:
+  http_name: tf_HTTP_MONITOR_%rand_int
+  https_name: tf_HTTPS_MONITOR_%rand_int
+  icmp_name: tf_ICMP_MONITOR_%rand_int
+  passive_name: tf_PASSIVE_MONITOR_%rand_int
+  tcp_name: tf_TCP_MONITOR_%rand_int
+  udp_name: tf_UDP-MONITOR_%rand_int
+acc:
+- config: |
+    lb_id = 1 
+    name  =  "$(http_name)"      
+    description  = "HTTP_MONITOR creating using tf"
+    type = "LBHttpMonitorProfile"
+    http_monitor {
+      fall_count = 8
+      interval =  10  
+      monitor_port = 50
+      rise_count = 5
+      timeout = 30
+      request_body = "request input body data"
+      request_method = "GET"
+      request_url = "https://request.com"
+      request_version  = "HTTP_VERSION_1_0" 
+      response_data = "Failed"
+      response_status_codes  = "500"
+    }
+  validations:
+    json.loadBalancerMonitor.monitorType: "LBHttpMonitorProfile"
+- config: |
+    lb_id = 1 
+    name  =  "$(https_name)"        
+    description  = "HTTPS_MONITOR creating using tf"
+    type = "LBHttpsMonitorProfile"
+    https_monitor {
+      fall_count = 3
+      interval =  5  
+      monitor_port = 80
+      rise_count = 3
+      timeout = 15
+      request_body = "request input body data"
+      request_method = "GET"
+      request_url = "https://test.com"
+      request_version  = "HTTP_VERSION_1_1" 
+      response_data = "success"
+      response_status_codes  = "201,200"
+    }
+  validations:
+    json.loadBalancerMonitor.monitorType: "LBHttpsMonitorProfile"
+- config: |
+    lb_id = 1 
+    name  =  "$(icmp_name)"        
+    description  = "ICMP_MONITOR update using tf"
+    type = "LBIcmpMonitorProfile"
+    icmp_monitor {
+      fall_count = 30
+      interval =  50 
+      monitor_port = 8
+      rise_count = 3
+      timeout = 15
+      data_length = 32
```

Comment:
Can you add at least one negative testcase ? Please refer instance testcase(template Error validation)

Reply:
done